### DISCUSSION TITLE
How do I get off the first page

### DISCUSSION THREAD

**Discussion by mrg9999-2024** [WerWolv/ImHex#1712] [Get Help]
Installed
run
How do I get off this page. I just want to edit some arbitary file
![image](https://github.com/WerWolv/ImHex/assets/170776190/95e2bf88-94e7-4fb0-bad9-8e1704cd0fb1)

**Comment by paxcut** [comment]
The small x on the upper left corner is for closing windows. The first page is actually the one behind that and from  there you can load files, etc...
